lib/utils/helpers.ts

```diff
@@ -16,16 +16,11 @@ limitations under the License.
 
 import type { InitializeEmitter, OperationState } from 'balena-device-init';
 import type * as BalenaSdk from 'balena-sdk';
-import { spawn, SpawnOptions } from 'child_process';
+
 import * as _ from 'lodash';
-import * as os from 'os';
-import type * as ShellEscape from 'shell-escape';
+import { promisify } from 'util';
 
-import type { Device, PineOptions } from 'balena-sdk';
-import { ExpectedError, SIGINTError } from '../errors';
 import { getBalenaSdk, getChalk, getVisuals } from './lazy';
-import { promisify } from 'util';
-import { isSubcommand } from '../preparser';
```

Comment:
I've done a cleanup of module-level imports in this file, so that there are fewer side effects (unnecessary cascading imports) of merely importing this file to use perhaps a single function like `safeUmount`.